x-pack/test/security_functional/plugins/test_endpoints/server/init_routes.ts

```diff
@@ -39,6 +39,32 @@ export function initRoutes(
   );
 
   const router = core.http.createRouter();
+
+  for (const isAuthFlow of [true, false]) {
+    router.get(
+      {
+        path: `/authentication/app/${isAuthFlow ? 'auth_flow' : 'not_auth_flow'}`,
+        validate: {
+          query: schema.object({
+            statusCode: schema.maybe(schema.number()),
+            message: schema.maybe(schema.string()),
+          }),
+        },
+        options: { tags: isAuthFlow ? ['security:authFlow'] : [] },
```

Comment:
__nit:__ it'd make sense to export `ROUTE_TAG_AUTH_FLOW` const and re-use it here instead of hardcoding `'security:authFlow'` string, [like we did for `ROUTE_TAG_CAN_REDIRECT`](https://github.com/elastic/kibana/blob/19bd682c8845a394f4f210bdd92cafed7e6a6314/x-pack/plugins/security/server/index.ts#L27)